More documentation content

dslosky-usgs opened the pull request:
Added content to the workbook page and a buffer for pyCast dev docs and updated .gitignore

dslosky commented:
The html files are changed in the first commit because I rebuilt the docs with the correct web address for the user/admin documentation. Now that the page will be rendered with Jekyll, we should just let Jekyll render the root directory... I'll make that change

Commit:
Add buffer page for pyCast

Commit:
cleaned up pycast directory

Commit:
Add configs to get pycast sidebar working

Commit:
Added basic aebm workbook info

Workbook page now also uses a sidebar and the sidebar for pycast was cleaned up

Commit:
Added some fac/group/user content to workbook page

Commit:
update .gitignore

Commit:
Update and merge conflics with Wan
